cds/webapi/C#/BlazorCDS/BlazorCDS/Program.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Net.Http;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Text;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using BlazorCDS.Models;
+
+namespace BlazorCDS
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var builder = WebAssemblyHostBuilder.CreateDefault(args);
+            builder.RootComponents.Add<App>("app");
+
+            builder.Services.AddTransient(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+
+            //Get configuration data about the Global Discovery API set in wwwroot/appsettings.json
+            var GDSWebApiConfig = builder.Configuration.GetSection("GDSWebAPI");
+            var gdsResourceUrl = GDSWebApiConfig.GetSection("ResourceUrl").Value;
+
+            // Get configuration data about the Web API set in wwwroot/appsettings.json
+            var CDSWebApiConfig = builder.Configuration.GetSection("CDSWebAPI");
+            var version = CDSWebApiConfig.GetSection("Version").Value;
+            var timeoutSeconds = int.Parse(CDSWebApiConfig.GetSection("TimeoutSeconds").Value);
+
+            // Create an named definition of an HttpClient that can be created in a component page
+            builder.Services.AddHttpClient("GDSClient", client =>
+            {
+                client.BaseAddress = new Uri(gdsResourceUrl);
```

Comment:
Could this:
`client.BaseAddress = new Uri(gdsResourceUrl);`
Simply be:
`client.BaseAddress = new Uri("https://globaldisco.crm.dynamics.com");`

This way we can skip storing and retrieving it in the appsettings

Reply:
kept the gdsResourceUrl because we're referencing this in the scope request as well, so don't want to repeat the GDS URL again in the same file. Should've made it a constant instead.